src/main/java/gov/cabinetoffice/gap/adminbackend/controllers/GrantMandatoryQuestionsController.java

```diff
@@ -0,0 +1,72 @@
+package gov.cabinetoffice.gap.adminbackend.controllers;
+
+import gov.cabinetoffice.gap.adminbackend.services.GrantMandatoryQuestionService;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.springframework.core.io.InputStreamResource;
+import org.springframework.http.ContentDisposition;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.io.*;
+
+import static gov.cabinetoffice.gap.adminbackend.controllers.SubmissionsController.EXPORT_CONTENT_TYPE;
+
+@Log4j2
+@RestController
+@RequestMapping("/mandatory-questions")
+@Tag(name = "Mandatory questions", description = "API for handling mandatory questions")
+@RequiredArgsConstructor
+public class GrantMandatoryQuestionsController {
+
+    final private GrantMandatoryQuestionService grantMandatoryQuestionService;
+
+    @GetMapping("/does-scheme-have-completed-mandatory-questions/{schemeId}")
+    public ResponseEntity<Boolean> doesSchemeHaveCompletedMandatoryQuestions(@PathVariable Integer schemeId) {
```

Comment:
This doesn't follow expected naming conventions in Java for methods that return boolean values. In general methods should start with `is` or `has`. 

It might seem a bit strange but more conventional names for this method might be `isCompleteBySchemeId` or `hasCompletedMandatoryQuestions`